src/client/components/TotalPriceCard/TotalPriceCard.styles.css

```diff
@@ -19,11 +18,11 @@
   text-align: right;
 }
 
-.wrapper h3 {
+.wrapperTotalPrice h3 {
   font-size: 1.3rem;
   padding-left: 1.2rem;
 }
 
-.wrapper-content .line {
+.wrapperContentTotalPrice .line {
   border-top: 1px solid lightgray;
```

Comment:
I personally prefer to add a border, rather than a line. You could use this piece of code and add it to the div you called text-rigth. But it is not an impediment to merging the PR, just a comment for next time :)

Reply:
Hi Valentina, 
I have tried to put the border under text-right, but the border comes on top of the amounts as well like in the image below:
![Screenshot 2021-12-08 at 23 09 04](https://user-images.githubusercontent.com/81102420/145292518-09ab220c-88ff-4cdb-9fda-a95d98ac4632.png)

So i think it is better to give the border seperately